### DISCUSSION TITLE
system monitor

### DISCUSSION THREAD

**Discussion by billykd4dpb** [linuxmint/discussions#392] [Questions]
i transitioned to mint and am trying to find a monitor that will display the CPU temp in the dock as a icon or something can anyone help me out

**Comment by rcalixte** [comment]
There is an applet called [CPU Temperature Indicator](https://cinnamon-spices.linuxmint.com/applets/view/106) that you should be able to install from the system settings. You can also search for alternatives or other categories like Desklets, Extensions, or Themes that fit your needs.

![Applets](https://github.com/linuxmint/discussions/assets/10281587/4d881a30-4505-4711-8a7c-36ba3d7685e5)
